app/mailers/user_mailer.rb

```diff
@@ -391,6 +391,16 @@ def suspended_reset_password
     end
   end
 
+  def gpo_reminder
+    with_user_locale(user) do
+      @date_letter_was_sent = I18n.l(
+        user.gpo_verification_pending_profile.gpo_verification_pending_at,
```

Comment:
@date_letter_was_sent was ambiguous when I read through the code, between USPS letter and email letter. How about @usps_letter_sent_at or @gpo_verification_pending_at (yeah, the gpo term is not great, but it matches the rest of the code).

Reply:
Changed to `@gpo_verification_pending_at`